### DISCUSSION TITLE
How to I add a submodule to GitHub pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23216] [Other Feature Feedback, Questions, & Ideas]
<p>Hello , How do I add a submodule to GitHub pages  so I can have a <a href="http://2500dog100.github.io/About-me" rel="noopener nofollow ugc">2500dog100.github.io/About-me</a><br>
My website is  <a href="https://2500dog100.github.io" rel="noopener nofollow ugc">2500dog100.github.io</a></p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! In the interest of clarity and precision, I don’t think what you are referring to is a “submodule”. With that said, you should create an “about-me.html” file and put whatever you want in it, then commit that file to the website repo. In ~15 minutes you should see the new page show up. (Note that you may need to add a trailing “/“ at the end of the url to see the page correctly)</p>

**Comment by ghost** [comment]
<p>That solved my problem</p>
